mismi-s3/src/Mismi/S3/Commands.hs

```diff
@@ -144,6 +145,13 @@ exists :: Address -> AWS Bool
 exists a =
   headObject a >>= pure . isJust
 
+existsPrefix :: Address -> AWS Bool
+existsPrefix (Address (Bucket b) (Key k)) =
+  fmap ((==) 1 . length . view A.lorsContents) . send $ A.listObjects (BucketName b)
```

Comment:
Might want to check `lorsCommonPrefixes` here as well, thinking of the nested prefix use case